packages/core/strapi/lib/core/domain/content-type/validator.js

```diff
@@ -54,18 +57,25 @@ const contentTypeSchemaValidator = yup.object().shape({
         for (const attrName in attributes) {
           const attr = attributes[attrName];
           if (attr.type === 'enumeration') {
-            // should not start by a number
-            if (attr.enum.some(startsWithANumber)) {
-              const message = `Enum values should not start with a number. Please modify your enumeration '${attrName}'.`;
+            const regressedValues = attr.enum.map(toRegressedEnumValue);
+
+            // should match the GraphQL regex
+            if (!regressedValues.every(value => GRAPHQL_ENUM_REGEX.test(value))) {
+              const message = `Invalid enumervarion value. Values should always have an alphabetical character preceding any number. Update your enumeration '${attrName}'.`;
```

Comment:
```suggestion
              const message = `Invalid enumeration value. Values should always have an alphabetical character preceding any number. Update your enumeration '${attrName}'.`;
```

+ same comment as above regarding the alphabetical character preceding "any" number

Reply:
Adressed in 1973fb57f6afd68de29492f1e21b850bfe0f8268